app/Utils/TestCreator.php

```diff
@@ -70,7 +72,15 @@ public function loadImage(Image $image): void
 
         // Decode the data
         $imgStr = base64_decode($image->Data);
-        $img = imagecreatefromstring($imgStr);
+        try {
+            $img = imagecreatefromstring($imgStr);
+        } catch (ErrorException) {
+            // Unable to create a valid image, substitute the broken image from CDash
+            Log::error("Unable to create image object from data in #{$this->testName}");
+            $image->Extension = 'image/png';
+            $img = imagecreatefromstring(strval(file_get_contents(public_path('/img/image_missing.png'))));
```

Comment:
After looking it over again, it seems like it would be better to just use [`imagecreatefrompng()`](https://www.php.net/manual/en/function.imagecreatefrompng.php) here.